src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const logo = document.querySelector('.logo');
+const succses = document.createElement('div');
+const error = document.createElement('div');
+
+succses.innerHTML = 'Promise was resolved!';
+error.innerHTML = 'Promise was rejected!';
+succses.classList.add('message');
+error.classList.add('error-message');
+body.append(succses);
+body.append(error);
+error.style.display = 'none';
+succses.style.display = 'none';
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve(succses.style.display = 'block');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    setTimeout(() => {
+      reject(new Error(error.style.display = 'block'));
+    }, 3000);
+  });
```

Comment:
Second promise should be rejected after 3 seconds. It should not depend on click on logo.